web/src/main/java/org/akaza/openclinica/service/ParticipantServiceImpl.java

```diff
@@ -156,8 +156,9 @@ public String createParticipant(SubjectTransferBean subjectTransfer,StudyBean cu
             studySubjectBean = this.getStudySubjectDao().createWithoutGroup(studySubjectBean);
 
         }
-        if(studySubjectBean!=null && !studySubjectBean.getStatus().equals(Status.AVAILABLE))
-            throw new OpenClinicaSystemException(ErrorConstants.ERR_PARTICIPANT_ID_NOT_AVAILABLE);
```

Comment:
instead of completely removing this check, maybe we should check for not available and not signed?

Reply:
code updated